### ISSUE TITLE
[Windows] ng g m doesn't work

### ISSUE THREAD

**Issue by jacobypl** [NativeScript/nativescript-schematics#91]
Hi,

``` ng g m moduleName ``` generates issues as below on Windows:

```
Schematic input does not validate against the Schema: {"routing":false,"routingScope":"Child","flat":false,"commonModule":true,"name":"test","project":"ngTestNative","path":"src\\app","spec":true}
Errors:

  Data path ".path" should match format "path".
```

When I changed to WSL (Ubuntu) everything is ok. Issue occurs with @nativescript/schematics. Standard ng new without schematic is working fine.

**Comment by sis0k0** [comment]
Hi @jacobypl,

This looks like the same problem in #85. Does it happen on Windows?

**Comment by sis0k0** [comment]
Fixed in `0.3.0-rc.0`
